bin/run-in-docker.sh

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env sh
+
+print_help() {
+  echo "Script to run docker containers for Spring Boot Template API service
+
+  Usage:
+
+  ./run-in-docker.sh [OPTIONS]
+
+  Options:
+    --clean, -c                   Clean and install current state of source code
+    --install, -i                 Install current state of source code
+    --param PARAM=, -p PARAM=     Parse script parameter
+    --help, -h                    Print this help block
+
+  Available parameters:
+
+  "
+}
+
+# script execution flags
+GRADLE_CLEAN=false
+GRADLE_INSTALL=false
+
+# TODO custom environment variables application requires.
```

Comment:
Ok for now as it's just first commit, but let's make sure we remove any TODOs

Reply:
It was already present in the template. I can remove if required

Reply:
Ok to leave for now as I said.